content/guides/concepts/pinning.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Pinning"
+menu:
+    guides:
+        parent: concepts
+---
+
+## Summary
+
+IPFS nodes treat the data they store like a cache, meaning that there is no guarantee that the data will continue to be stored.  Pinning a CID tells an IPFS server that the data is important and mustn't be thrown away.
+
+You should pin any content you consider important, to ensure that content is retained long-term. Since data important to someone else may not be important to you, pinning lets you have control over the disk space used by your IPFS nodes.
+
+## Context 
+
+An IPFS node can store data based on different kinds of user events.  For instance, a person may add a file with `ipfs add ...`.  Or a piece of unknown IPFS content can be requested, and the node can consult with its peers to find this data - the result will be stored in the local cache.  These data can use different policies for managing the cache - `ipfs add ...` will automatically pin the content.  Other IPFS commands do not include automatic pinning.
+
+When garbage-collection is triggered on a node, any pinned content is automatically exempt from deletion.  Non-pinned data is otherwise eligible to be deleted to save space (if requested, this data can be re-retrieved from another node. 
```

Comment:
“Garbage collection” should not be hyphenated.

> is otherwise eligible to be deleted

I think this could be simplified and made easier to read by just saying “may be deleted”.

Also, the closing parenthesis at the end is missing.